lib/fog/mock/errors.rb

```diff
@@ -0,0 +1,5 @@
+module Fog
+  module Errors
+    class MockNotImplemented < Fog::Errors::Error; end
+  end
+end
```

Comment:
Still. There is only one constant being defined and the file name does not follow it.
It is a good practice to make the path follow the constant name to make it easier to find a class in the project.